### ISSUE TITLE
POLL: What should be added to Nextcloud Music next?

### ISSUE THREAD

**Issue by ei8fdb** [owncloud/music#881]
What features do you see as important/useful to add to Nextcloud Music?

You can give your input with this short poll:

https://saneuxdesign.survey.fm/what-features-should-be-added-to-nextcloud-owncloud-music-app-next

You have the opportunity to add your own feature requests if it’s not listed there.

(The background: I’ve been making some small UX contributions to the Music app. I asked Pauli if a short poll to understand what features people saw as important would be helpful and he’s said yes. So here it is. :slight_smile: )

(This is obviously not an issue - I'm posting this here as many Music users come to the repo to for feature requests. If this isn't appropriate, please do close it. Apologies if so!)

**Comment by jecogeo** [comment]
Is it possiblem to share the partial results of this poll?

**Comment by paulijar** [comment]
@jecogeo Actually @ei8fdb was planning to close the poll soon, so I'm sure we'll get the final results in the near future.

**Comment by ei8fdb** [comment]
> @jecogeo Actually @ei8fdb was planning to close the poll soon, so I'm sure we'll get the final results in the near future.

@jecogeo Certainly, I'll be doing a blogpost with the results. I'll post it here, and also in the Nextcloud thread I opened. What actually happens in the end will be up to @paulijar as he's the maintainer, but I'll give my opinion for what its worth. :)

**Comment by facorazza** [comment]
Any news on the results?
